lib/Client.js

```diff
@@ -569,6 +664,28 @@ class Client extends EventEmitter {
         }).then((template) => new GuildTemplate(template, this));
     }
 
+    /**
+    * Respond to the interaction with a message
+    * Note: Use webhooks if you have already responded with an interaction response.
+    * @arg {String} interactionID The interaction ID.
+    * @arg {String} interactionToken The interaction Token.
+    * @arg {Object} options The options object.
+    * @arg {Object} [options.data] The data to send with the response.
+    * @arg {Object} [options.data.allowedMentions] A list of mentions to allow (overrides default)
+    * @arg {Boolean} [options.data.allowedMentions.everyone] Whether or not to allow @everyone/@here.
+    * @arg {Boolean | Array<String>} [options.data.allowedMentions.roles] Whether or not to allow all role mentions, or an array of specific role mentions to allow.
+    * @arg {Boolean | Array<String>} [options.data.allowedMentions.users] Whether or not to allow all user mentions, or an array of specific user mentions to allow.
+    * @arg {String} [options.data.content] A content string
+    * @arg {Object} [options.data.embed] An embed object. See [the official Discord API documentation entry](https://discord.com/developers/docs/resources/channel#embed-object) for object structure
+    * @arg {Boolean} [options.data.flags] 64 for Ephemeral (Applies to Slash Commands and Message Components)
+    * @arg {Boolean} [options.data.tts] Set the message TTS flag
+    * @arg {Number} options.type The response type to send [Check Discord docs for valid responses](https://discord.com/developers/docs/interactions/slash-commands#interaction-response-interactioncallbacktype).
+    * @returns {Promise}
+    */
+    createInteractionResponse(interactionID, interactionToken, options) {
+        return this.requestHandler.request("POST", Endpoints.INTERACTION_RESPOND(interactionID, interactionToken), true, options);
```

Comment:
To make the 3 line+ log spam these cause, this should be added
```suggestion
        return this.requestHandler.request("POST", Endpoints.INTERACTION_RESPOND(interactionID, interactionToken), true, options, undefined, "/interactions/:id/:token/callback");
```
(also can you not send files through interactions? this doesn't have a way to provide them)

Reply:
You cannot yet, tho you can after the initial response

Reply:
ah, no wonder I've never had an issue, I've only done them in followups